Dockerfile

```diff
@@ -51,7 +51,7 @@ ARG TAG
 WORKDIR /
 RUN git clone -b ${KUBERNETES_VERSION} --depth=1 https://github.com/kubernetes/kubernetes.git
 WORKDIR /kubernetes
-RUN make KUBE_GIT_VERSION=${TAG} WHAT='cmd/kube-apiserver cmd/kube-controller-manager cmd/kube-proxy cmd/kube-scheduler cmd/kubeadm cmd/kubectl cmd/kubelet vendor/k8s.io/apiextensions-apiserver'
+RUN make KUBE_GIT_VERSION=${TAG} GOLDFLAGS='-extldflags "-static"' WHAT='cmd/kube-apiserver cmd/kube-controller-manager cmd/kube-proxy cmd/kube-scheduler cmd/kubeadm cmd/kubectl cmd/kubelet vendor/k8s.io/apiextensions-apiserver'
```

Comment:
We have some extra logic around setting cflags in build-binary - do we need to add that here, or are we OK just adding the static ldflag for now?

Reply:
I think this is fine for now until it becomes necessary to pass more flags for the linker.